File: NotVerySecure.py
def alphanumeric(string):
    print(string)
    if string.strip() == "":
        return False

    checker = set("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890")
    for char in string:
        if char not in checker:
            return False

    return True

File: NotVerySecure.js
function alphanumeric(string){
    console.log(string)
    if(string.trim() == ""){
      return false
    }
    
    const checker = [..."abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890"]
    for(let i = 0; i < string.length; i++){
        if(!checker.includes(string[i])){
            return false
        }
    }
    
    return true
  }